docs/deployed-contracts/mainnet-lido-v2-proposed.md

```diff
@@ -0,0 +1,56 @@
+# Proposed Lido V2 Addresses
+
+## Lido V2 Core Protocol
+
+- (proposed) LidoLocator: [`0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb`](https://etherscan.io/address/0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb) (proxy)
+  - proposed implementation: [`0x1D920cc5bACf7eE506a271a5259f2417CaDeCE1d`](https://etherscan.io/address/0x1D920cc5bACf7eE506a271a5259f2417CaDeCE1d)
+- Lido and stETH token: [`0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84`](https://etherscan.io/address/0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84) (proxy)
+  - proposed implementation: [`0x17144556fd3424EDC8Fc8A4C940B2D04936d17eb`](https://etherscan.io/address/0x17144556fd3424EDC8Fc8A4C940B2D04936d17eb)
+- wstETH token: [`0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0`](https://etherscan.io/address/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0)
+- (proposed) EIP-712 helper for stETH: [`0x8F73e4C2A6D852bb4ab2A45E6a9CF5715b3228B7`](https://etherscan.io/address/0x8F73e4C2A6D852bb4ab2A45E6a9CF5715b3228B7)
+- (proposed) StakingRouter: [`0xFdDf38947aFB03C621C71b06C9C70bce73f12999`](https://etherscan.io/address/0xFdDf38947aFB03C621C71b06C9C70bce73f12999) (proxy)
+  - proposed implementation: [`0xD8784e748f59Ba711fB5643191Ec3fAdD50Fb6df`](https://etherscan.io/address/0xD8784e748f59Ba711fB5643191Ec3fAdD50Fb6df)
+- Node Operators registry: [`0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5`](https://etherscan.io/address/0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5) (proxy)
+  - proposed implementation: [`0x8538930c385C0438A357d2c25CB3eAD95Ab6D8ed`](https://etherscan.io/address/0x8538930c385C0438A357d2c25CB3eAD95Ab6D8ed)
+- (proposed) Deposit Security module: [`0xC77F8768774E1c9244BEed705C4354f2113CFc09`](https://etherscan.io/address/0xC77F8768774E1c9244BEed705C4354f2113CFc09)
+- Execution Layer Rewards Vault: [`0x388C818CA8B9251b393131C08a736A67ccB19297`](https://etherscan.io/address/0x388C818CA8B9251b393131C08a736A67ccB19297)
+- (proposed) WithdrawalQueueERC721: [`0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1`](https://etherscan.io/address/0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1) (proxy)
+  - proposed implementation: [`0xE42C659Dc09109566720EA8b2De186c2Be7D94D9`](https://etherscan.io/address/0xE42C659Dc09109566720EA8b2De186c2Be7D94D9)
+- WithdrawalVault: [`0xb9d7934878b5fb9610b3fe8a5e441e8fad7e293f`](https://etherscan.io/address/0xb9d7934878b5fb9610b3fe8a5e441e8fad7e293f) (proxy)
+  - proposed implementation: [`0xCC52f17756C04bBa7E377716d7062fC36D7f69Fd`](https://etherscan.io/address/0xCC52f17756C04bBa7E377716d7062fC36D7f69Fd)
+- (proposed) Burner of stETH: [`0xD15a672319Cf0352560eE76d9e89eAB0889046D3`](https://etherscan.io/address/0xD15a672319Cf0352560eE76d9e89eAB0889046D3)
```

Comment:
Strange to see `of stETH` here. Let's stick to contract names. Or add a short description separately if required.
```suggestion
- (proposed) Burner: [`0xD15a672319Cf0352560eE76d9e89eAB0889046D3`](https://etherscan.io/address/0xD15a672319Cf0352560eE76d9e89eAB0889046D3)
```